lib/api/controllers/authController.js

```diff
@@ -109,13 +131,13 @@ class AuthController {
     assertBodyHasAttribute(request, 'token');
 
     return this.kuzzle.repositories.token.verifyToken(request.input.body.token)
-      .then(token => Promise.resolve({valid: true, expiresAt: token.expiresAt}))
+      .then(token => Bluebird.resolve({valid: true, expiresAt: token.expiresAt}))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 81 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)